data/wanderer outfits.txt

```diff
@@ -250,15 +250,15 @@ outfit "Thunderhead Launcher"
 		"infrared tracking" .9
 		"radar tracking" .6
 		"missile strength" 12
-	description "A prime example of Wanderer ingenuity, Thunderhead missiles each carry five submunitions that are capable of independent tracking. Even if the target is able to shoot down or evade a few of them, the remaining missile fragments will still find their mark."
+	description "A prime example of Wanderer ingenuity, Thunderhead missiles each carry five submunitions that are capable of independent tracking. Even if the target is able to shoot down or evade a few of them, the remaining missile fragments will still find their mark. Each launcher's internal magazine can hold five missiles."
```

Comment:
>In general, I try to avoid ship and outfit descriptions that do nothing but reiterate the information you can get from the weapon's stats. It's much better for the descriptions to provide some sort of flavor text (historical information, etc.) that is interesting or funny in its own right.